code/modules/mob/living/silicon/ai/say.dm

```diff
@@ -129,7 +129,8 @@ var/const/VOX_PATH = "sound/vox_fem/"
 
 	announcing_vox = world.time + VOX_DELAY
 
-	log_game("[key_name_admin(src)] made a vocal announcement with the following message: [message].")
+	log_game("[key_name(src)] made a vocal announcement: [message].")

```

Comment:
wouldn't it be better to use key_name_log for this?

Reply:
Perhaps. Though looking into it, all it offers over doing key_name is checks if they're an antag or not or if they're SSD or not. But the IsAntag check only works on carbons.. which is weird. And the AI can't make an announcement if it's SSD so basically swapping this out gave me an identical log.